SwiftyJSONAccelerator/SJModelGenerator.swift

```diff
@@ -189,9 +211,75 @@ public class ModelGenerator {
             content = content.stringByReplacingOccurrencesOfString("{OBJECT_KIND}", withString: type)
             content = content.stringByReplacingOccurrencesOfString("{STRING_CONSTANT_BLOCK}", withString: stringConstants)
             content = content.stringByReplacingOccurrencesOfString("{PROPERTIES}", withString: declarations)
-            content = content.stringByReplacingOccurrencesOfString("{INITALIZER}", withString: initalizers)
-            content = content.stringByReplacingOccurrencesOfString("{ENCODERS}", withString: encoders)
-            content = content.stringByReplacingOccurrencesOfString("{DECODERS}", withString: decoders)
+            
+            if self.supportNSCoding
+            {
+                content = content.stringByReplacingOccurrencesOfString("{NSCODING_PROTOCOL_SUPPORT}", withString: ", NSCoding")
+                content = content.stringByReplacingOccurrencesOfString("{NSCODING_SUPPORT}", withString: "\t// MARK: NSCoding Protocol\n\trequired public init(coder aDecoder: NSCoder) {\n\t{DECODERS}\n\t}\n\n\tpublic func encodeWithCoder(aCoder: NSCoder) {\n{ENCODERS}\n\t}")
```

Comment:
Probably should. I modified that part before I created template files for SwiftyJSON / ObjectMapper.